portal/pom.xml

```diff
@@ -59,6 +59,10 @@
 			<artifactId>com.liferay.util.taglib</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.liferay</groupId>
+			<artifactId>com.liferay.journal.api</artifactId>
```

Comment:
Ooops you're right. Doesn't make sense to make users download this when compiling.